daemon/daemon.go

```diff
@@ -980,6 +985,10 @@ func initNetworkController(config *Config) (libnetwork.NetworkController, error)
 	return controller, nil
 }
 
+func (daemon *Daemon) NetworkApiRouter() func(w http.ResponseWriter, req *http.Request) {
```

Comment:
This doesn't look right. Does this need writer and request? Above where this is called nothing is being sent.
Also this is importing `net/http` into daemon which doesn't seem right.